frontend/src/app/api/auth/verification/check/route.ts

```
@@ -0,0 +1,69 @@
+
+    const responseJson = await response.json();
+    // check if status is 200
+    // save tokens into a variable
+    // delete tokens from response
+    // set tokens in cookies
+    // send response
+    if (response.status === 200) {
+        const accessToken = responseJson.data.tokens.access;
+        const refreshToken = responseJson.data.tokens.refresh;
+        // delete tokens from response
+        delete responseJson.data.tokens;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)